mkosi/resources/mkosi.md

```diff
@@ -350,6 +352,34 @@ Architecture=|x86-64
 Architecture=|arm64
 ```
 
+The `[TriggerMatch]` section can be used to indicate triggering match
+sections. These are identical to triggering conditions except they apply
+to the entire match section instead of just a single condition. As an
+example, the following will match if the distribution is `debian` and
+the release is `bookworm` or if the distribution is `ubuntu` and the
+release is `focal`.
+
+```conf
+[TriggerMatch]
+Distribution=debian
+Release=bookworm
+
+[TriggerMatch]
+Distribution=ubuntu
+Release=focal
+```
+
+The semantics of conditions in `[TriggerMatch]` sections is the same as
+in `[Match]`, i.e. all normal conditions are joined by a logical AND and
+all triggering conditions are joined by a logical OR. When mixing
+`[Match]` and `[TriggerMatch]` sections, a match is achieved when all
+`[Match]` sections match and at least one `[TriggerMatch]` section
+matches. No match sections are valued as true. Logically this means:
+
+```
+(∧ᵢ Matchᵢ) ∧ (∨ᵢ TriggerMatchᵢ)
```

Comment:
Usually one would use the bigger n-ary operators here (⋀ and ⋁) for the expressions in the parentheses.

We could also write this as a proper math section which would be rendered prettier in the markdown preview. I tried this out put sadly the roff output seems to have underlines then - no clue what is going on there:

![Rendered man page](https://github.com/systemd/mkosi/assets/24257556/12afb994-315b-4b64-b6fc-f7fd11eca616)

![Rendered markdown](https://github.com/systemd/mkosi/assets/24257556/3d64a0de-1df9-4163-8510-c4018c684772)

I just tried out several combinations be the underlines and tildes seem to always be generated by pandoc...

```md
$$
(\wedge_i Match_i) \wedge (\bigvee_i \text{TriggeringMatch}_i)
$$

$$
(⋀_\text{ᵢ} \text{Match}_{i}) ∧ (⋁ᵢ TriggerMatch_i)
$$

```txt
(⋀ᵢ Matchᵢ) ⋀ (⋁ᵢ TriggerMatchᵢ)
```
```

Reply:
I went for working in the confines of unicode, since that should work in a man page ;) You're right though, that I missed `N-ARY LOGICAL AND` (and or) 
```
(⋀ᵢ Matchᵢ) ∧ (⋁ᵢ TriggerMatchᵢ)
```
they were too small on my screen to see the difference.

Reply:
```suggestion
(⋀ᵢ Matchᵢ) ∧ (⋁ᵢ TriggerMatchᵢ)
```